src/status_im/contacts/contact_list_modal/views.cljs

```diff
@@ -0,0 +1,78 @@
+(ns status-im.contacts.contact-list-modal.views
+  (:require-macros [status-im.utils.views :refer [defview letsubs]])
+  (:require [re-frame.core :refer [dispatch]]
+            [status-im.components.common.common :as common]
+            [status-im.components.renderers.renderers :as renderers]
+            [status-im.components.react :refer [view list-view list-item]]
+            [status-im.components.contact.contact :refer [contact-view]]
+            [status-im.components.action-button.action-button :refer [action-button
+                                                                      action-separator]]
+            [status-im.components.action-button.styles :refer [actions-list]]
+            [status-im.components.status-bar :refer [status-bar]]
+            [status-im.components.toolbar-new.view :refer [toolbar-with-search]]
+            [status-im.components.drawer.view :refer [drawer-view]]
+            [status-im.contacts.styles :as st]
+            [status-im.utils.listview :as lw]
+            [status-im.i18n :refer [label]]))
+
+(defview contact-list-modal-toolbar []
+  (letsubs [show-search [:get-in [:toolbar-search :show]]
```

Comment:
The thing is that adding of another layer of abstraction here requires a lot of boilerplate glue code and in most of cases it's useless. We had discussed this moew then once and my opinion is the next:
1) `:get` subscriptions are looking for the map's root elements, and in most cases these element don't change over time
2) `:get-in` paths are more mutable and we can agree that if such subscription should be used in more than one ns better to write wrapper